web/src/app/schedules/ScheduleRuleForm.js

```diff
@@ -100,131 +99,13 @@ const query = gql`
     }
   }
 `
+const useStyles = makeStyles(styles)
```

Comment:
lets add the styles object directly here, since `styles` isn't used anywhere else

```
const useStyles = makeStyles({
  noPadding: {
    padding: 0,
  },
  . . .
})
```